src/utils/network-monitor.ts

```diff
@@ -582,6 +585,8 @@ export class NetworkMonitor {
       this.log(`📄 Interfaces address: ${this.interfacesAddress}`)
       this.log(`📄 Operator address: ${this.operatorAddress}`)
       this.log(`📄 Registry address: ${this.registryAddress}`)
+      this.log(`📄 HLG Token address: ${this.tokenAddress}`)
+      this.log(`📄 CXIP ERC721 address: ${this.cxipERC721Address}`)
```

Comment:
Since we are using this address just so we can have a hardhat class, and is really an implementation detail. Should we label it? I don' t want people thinking that there is only cxipERC721Address. How do we make sure people don't make the wrong assumption?

1. `CXIP ERC721 address (internal instance)` 
2. `CXIP ERC721 address (internal)`
3. `CXIP ERC721 address (hardhat contract initializer)`

🤔

Reply:
oh yeah that's true. we don't need to log it here. it was mostly just for me while i was testing this